app/src/main/java/chat/rocket/android/util/HttpLoggingInterceptor.kt

```diff
@@ -0,0 +1,307 @@
+package chat.rocket.android.util
+
+import okhttp3.Headers
+import okhttp3.Interceptor
+import okhttp3.MediaType
+import okhttp3.OkHttpClient
+import okhttp3.Response
+import okhttp3.internal.http.HttpHeaders
+import okhttp3.internal.platform.Platform
+import okhttp3.internal.platform.Platform.INFO
+import okio.Buffer
+import okio.GzipSource
+import java.io.EOFException
+import java.io.IOException
+import java.nio.charset.Charset
+import java.util.concurrent.TimeUnit
+
+/**
+ * An OkHttp interceptor which logs request and response information. Can be applied as an
```

Comment:
documentation...
💯

Reply:
💯 copied from OkHttp logger... It is just a modified/kotlinfied version to remove the auth token from the logs